Dockerfile

```diff
@@ -71,6 +71,9 @@ RUN --mount=type=bind,target=. \
   xx-go install std
   xx-go build -ldflags "$(cat /tmp/.ldflags)" -o /out/docker-credential-osxkeychain-${TARGETARCH}${TARGETVARIANT} ./osxkeychain/cmd/main_darwin.go
   xx-verify /out/docker-credential-osxkeychain-${TARGETARCH}${TARGETVARIANT}
+
+  xx-go build -ldflags "$(cat /tmp/.ldflags)" -o /out/docker-credential-pass-${TARGETARCH}${TARGETVARIANT} ./pass/cmd/main.go
+  xx-verify /out/docker-credential-pass-${TARGETARCH}${TARGETVARIANT}
```

Comment:
should add also `${TARGETOS}` now that is not only linux.

Reply:
Ah, good one; wasn't sure if we collected them all in the same spot.

Let me have a look (we probably still need to look if we want to enable this by default and/or if that could mess up release pipelines elsewhere)